File: lua/fizzbuzz.lua
for i = 1, 100 do
  fizz = ""
  buzz = ""

  if i % 3 == 0 then
    fizz = "fizz"
  end

  if i % 5 == 0 then
    buzz = "buzz"
  end

  if fizz~="" or buzz~="" then
    print(fizz..buzz)
  else
    print(i)
  end
end

File: c/fizzbuzz.c
#include <stdio.h>

int main(void)
{
  for (int i = 1; i <= 100; ++i) {

    int modThree = i % 3;
    int modFive = i % 5;

    if(!modThree && !modFive) {
      printf("fizzbuzz");
    } else if (!modThree) {
      printf("fizz");
    } else if (!modFive) {
      printf("buzz");
    } else {
      printf("%d", i);
    }

    printf("\n");
  }
}
